### ISSUE TITLE
Button to centre map on current location (feature request)

### ISSUE THREAD

**Issue by Boran** [soaringmeteo/soaringmeteo#77]
On several maps, there is a button to recentre the map on the current location (useful when called from phones).

Would that be possible on v2, is it complicated?

**Comment by julienrf** [comment]
Yes it is possible. Do you have an idea of design in mind? Should the button be always visible? Like in https://spotair.mobi:

![image](https://user-images.githubusercontent.com/332812/235512138-25ca6960-2457-47b8-b9c4-34f111659fc4.png)

Or, should it be a button available in a menu? (good idea if the action is not used often, because it makes the UI less cluttered)

**Comment by julienrf** [comment]
I will think about how to include the “center the map on my location” button in the UI.

> Perhaps the location could be stored too (cookie)? and restored when one comes back to the page?

That is already the case. Is it not working for you?

**Comment by Boran** [comment]
Ah, I didn't check :-)

**Comment by julienrf** [comment]
Screencast showing the feature:

https://github.com/user-attachments/assets/295f0cc3-980f-4b3c-b792-e3d633df3042

**Comment by Boran** [comment]
Thanks!
